fdmi/plugins/fdmi_app

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/env python3
```

Comment:
I am thinking removing this python program completely.

This is not necessary for a demo.
The `fdmi_sample_plugin` just output the events to stdout.
That's enough to demo.

No need to process it.
We just need to keep this sample application as simple as possible.

Reply:
Yes. Remove the python file completely.
This is out of scope of a FDMI demo app.
The FDMI demo app just prints the received events into stdout.
Users can do anything about the output.
I know that the python program process this output from demo app in somewhat way.
But that is not the purpose of this demo app.

Reply:
I think this needs to be discussed with @kaustubh-d. You don't have to run the python program is you don't want but it is important demo aspect how the python can be used with it and since we build the part of the system on python I vote for keeping it.

Reply:
OK